packages/internal/src/files.ts

```diff
@@ -7,7 +7,7 @@ import { getPaths } from '@redwoodjs/project-config'
 import { getNamedExports, hasDefaultExport, fileToAst } from './ast'
 
 export const findCells = (cwd: string = getPaths().web.src) => {
-  const modules = fg.sync('**/*Cell.{js,jsx,ts,tsx}', {
+  const modules = fg.sync('web/src/**/*Cell.{js,jsx,ts,tsx}', {
```

Comment:
Because `cwd` is already /web/src (by default), I think this was actually correct the way it was originally written. Same goes for every other change, except for the one in `findDirectoryNameModules` because that one starts at the project base directory

Reply:
Fixed. All tests pass now.